src/Sylius/Bundle/CoreBundle/Resources/config/services.xml

```diff
@@ -237,6 +237,7 @@
             <argument type="service" id="sylius.factory.order_sequence" />
             <argument type="service" id="sylius.manager.order_sequence" />
         </service>
+        <service id="sylius.number_generator.sequential_order" alias="sylius.sequential_order_number_generator" />
```

Comment:
Consider this: is it set up correctly? The service with the same ID is in the OrderBundle, so I added the same alias in both places.

Reply:
I guess the one in OrderBundle should be enough